languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/disambiguation.xml

```diff
@@ -2171,6 +2171,26 @@ Copyright © 2013 Markus Brenneis, Daniel Naber, Jan Schreiber
     <!-- ============================================================================================================== -->
     <!-- IGNORE SPELLING -->
 
+    <rulegroup name="Ignore surnames after 'Sehr geehrte/r Frau/Herr'" id="SEHR_GEEHRTE_FRAU_X"><!--Sehr geehrte Frau Schnibbliduppsi.-->
```

Comment:
Leider finden wir dann auch Fehler nicht mehr, wenn jemand z.B. "Mller" statt "Müller" schreibt. Ich denke, da brauchen wir eine bessere Lösung...

Reply:
Wäre diese Regel zumindest für lange Namen eine Idee, sagen wir ab ca. 9 Buchstaben? 
Bsp: 

> Schmithausen
> Schmitzer
> Schmittlein
> Schmitendorf
> Schmitzler
> Schmitgen
> Schmitten
> Schmitmann
> Schmittenberg
> Schmithoff
> Schmittgens
> Schmittenhofen
> Schmithüsen

Reply:
Auch da kann man sich ja vertippen. Wenn wir eine lange Liste mit vielen Nachnamen haben, könnten wir die vielleicht nur in diesem Kontext ignorieren (statt sie einfach der `spelling.txt` hinzuzufügen)?

Reply:
Ok, danke, dann schließe ich diesen PR vorerst.